pkgs/games/anki/default.nix

```diff
@@ -158,6 +161,9 @@ buildPythonApplication rec {
       cp -rv anki aqt web $pp/
 
       wrapPythonPrograms
+      for program in $out/bin/*; do
```

Comment:
Why do we have to do `wrapPythonPrograms` in `buildPythonApplication`? I thought that'd happen automatically.

Also by doing this we get two wrappers, maybe give `makeWrapperArgs` `"\${qtWrapperArgs[@]}"`?